### ISSUE TITLE
Portal Stations:  Logging out and I lose all my portals...

### ISSUE THREAD

**Issue by Snayjay** [RustyMods/PortalStations#8]
I don't physically lose them, they are all still there.  But they are no longer tied and I can't see them in the list of portals.  So I have to run back to the portal... break it and build it in order to see it in the list.  I've logged out twice and both times it happens.  I have a large list of mods being used, but the only one that directly affects portals is Portal Everything.  I disabled Portal Everything since Portal Stations basically does everything it does.  And still no Portals in my list.

**Comment by Snayjay** [comment]
I thought maybe the problem was it's supposed to be loaded server side as well.  So I installed the .dll file to my server and relogged.  No portals in the list, so I connected 2 portals and logged out and back in... and still not saving.

**Comment by RustyMods** [comment]
odd - yes it needs to be on server to work.

**Comment by RustyMods** [comment]
These are portals you've built right ? Cause if it's one of your alt characters - you will need to make portal public

**Comment by Snayjay** [comment]
Yes, currently I'm the only person in the server.  I'm setting up a hosted server to host a heavily modded playthru and am currently setting it up.  I have the .dll installed in the plugins folder on my server and installed it on my client thru the Thunderstore Mod Manager.  I can give you a list of the mods I use if you think it will help... but like I said, I removed Portal Everything which was the one I thought would be conflicting the most.  Valheim Plus has some portal settings I used as well... I can go turn those off if you think that matters.   APPRECIATE you taking the time to help.  And yes, when I make a portal I set it to public before leaving.  However I've also clicked that option when opening a portal because I wasn't sure if that was a setting or part of the filter system.
